base/Dockerfile

```diff
@@ -41,6 +41,8 @@ ARG KOPANO_CORE_VERSION=newest
 ARG KOPANO_REPOSITORY_FLAGS="trusted=yes"
 ARG KOPANO_WEBAPP_REPOSITORY_URL="file:/kopano/repo/webapp"
 ARG KOPANO_WEBAPP_VERSION=newest
+ARG KOPANO_SMIME_REPOSITORY_URL="file:/kopano/repo/smime"
```

Comment:
Since there is also a separate repo/download for mdm and files I would suggest to rather name this variable `KOPANO_WEBAPP_SMIME_REPOSITORY_URL`